core/src/main/java/org/apache/calcite/runtime/SqlFunctions.java

```diff
@@ -123,6 +129,39 @@
   private SqlFunctions() {
   }
 
+  /** SQL TO_BASE64(string) function. */
+  public static String toBase64(String string) {
+    return toBase64_(string.getBytes(UTF_8));
+  }
+
+  /** SQL TO_BASE64(string) function for binary string. */
+  public static String toBase64(ByteString string) {
+    return toBase64_(string.getBytes());
+  }
+
+  private static String toBase64_(byte[] bytes) {
+    String base64 = Base64.getEncoder().encodeToString(bytes);
+    StringBuilder str = new StringBuilder(base64.length() + base64.length() / 76);
+    if (base64.length() > 76) {
```

Comment:
Do you really need this check ? Splitter will probably create single element iterator anyway.

Reply:
sorry，I didn't notice that，I have already addressed.